src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```diff
@@ -1,41 +1,51 @@
 import { useState, useMemo } from 'react'
 import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
-
+import AppChipList from '~/components/app-chips-list/AppChipList'
 import AppAutoComplete from '~/components/app-auto-complete/AppAutoComplete'
 import { categoriesMock } from '~/containers/tutor-home-page/subjects-step/constants'
 import { useTranslation } from 'react-i18next'
-
 import { styles } from '~/containers/tutor-home-page/subjects-step/SubjectsStep.styles'
 import img from '~/assets/img/tutor-home-page/become-tutor/study-category.svg'
 import AppButton from '~/components/app-button/AppButton'
 
 const SubjectsStep = ({ btnsBox }) => {
   const { t } = useTranslation()
-  const [subjects, setSubjects] = useState({
-    category: null,
-    subject: null
-  })
+  const [subjects, setSubjects] = useState({ category: null, subject: null })
+  const [listOfItems, setListOfItems] = useState([])
+  const [sameSubjectError, setSameSubjectError] = useState('')
```

Comment:
Better to save in state `true`/`false` value.
And in fact we don't need this state.
```suggestion
const sameSubjectError = listOfItems.includes(subjects.subject);
```

Reply:
I understand, thank you.